src/mame/sinclair/tsconf.h

```diff
@@ -13,11 +13,10 @@
 #include "spec128.h"
 
 #include "beta_m.h"
-
+#include "bus/spectrum/zxbus.h"
 #include "glukrs.h"
 #include "machine/pckeybrd.h"
 #include "machine/spi_sdcard.h"
-#include "neogs.h"
 #include "tsconfdma.h"
```

Comment:
You should keep local header before library headers – glukrs.h and tsconfdma.h should come before the headers from src/devices.